cd/nodeForIO.wsdl

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@corvettafabio if i remember correctly the name of the folder indicates the services offered by the function for others. Specifically the `cd` folder contains service `cdInfoWisp` called by NODO. 
If it's right, wouldn't it be better to put the new `nodoForIO.*` files in the `nodo` folder?

Reply:
You're right, but I think it's better to separate PSP/EC interface to IO interface.